docs/application/tizen-studio/index.md

```diff
@@ -9,7 +9,7 @@ In the Tizen Studio, you can perform all the same development activities that we
 
 To develop your applications, use the set of tools provided by the Tizen Studio:
 
-- [Setup](setup/installing-sdk.md)  
+- [Setup](setup/install-studio.md)  
```

Comment:
It looks like `install-sdk.md` is correct.

Reply:
done